internal/resourcestore/resourcecleaner.go

```diff
@@ -5,56 +5,68 @@ import (
 	"time"
 
 	"github.com/pkg/errors"
+	errorUtils "k8s.io/apimachinery/pkg/util/errors"
 	"k8s.io/apimachinery/pkg/util/wait"
 
 	"github.com/cri-o/cri-o/internal/log"
 )
 
-// A CleanupFunc is a function that cleans up one piece of
-// the associated resource.
-type CleanupFunc func() error
-
 // ResourceCleaner is a structure that tracks
 // how to cleanup a resource.
 // CleanupFuncs can be added to it, and it can be told to
 // Cleanup the resource
 type ResourceCleaner struct {
-	ctx   context.Context
-	funcs []CleanupFunc
+	funcs []cleanupFunc
 }
 
+// A cleanupFunc is a function that cleans up one piece of
+// the associated resource.
+type cleanupFunc func() error
+
 // NewResourceCleaner creates a new ResourceCleaner
-func NewResourceCleaner(ctx context.Context) *ResourceCleaner {
-	return &ResourceCleaner{
-		ctx:   ctx,
```

Comment:
An earlier commit from this PR adds ctx to `ResourceCleaner`, and now you remove it. It's better to fix the first commit right away (less changes in history, easier to review).